pallets/rc-migrator/src/lib.rs

```diff
@@ -575,6 +596,39 @@ pub mod pallet {
 					}
 				},
 				MigrationStage::NomPoolsMigrationDone => {
+					Self::transition(MigrationStage::FastUnstakeMigrationInit);
+				},
+				MigrationStage::FastUnstakeMigrationInit => {
+					Self::transition(MigrationStage::FastUnstakeMigrationOngoing {
+						next_key: None,
+					});
+				},
+				MigrationStage::FastUnstakeMigrationOngoing { next_key } => {
+					let res = with_transaction_opaque_err::<Option<_>, Error<T>, _>(|| {
+						match FastUnstakeMigrator::<T>::migrate_many(next_key, &mut weight_counter)
+						{
+							Ok(last_key) => TransactionOutcome::Commit(Ok(last_key)),
+							Err(e) => TransactionOutcome::Rollback(Err(e)),
+						}
+					})
+					.expect("Always returning Ok; qed");
+
+					match res {
+						Ok(None) => {
+							Self::transition(MigrationStage::FastUnstakeMigrationDone);
+						},
+						Ok(Some(next_key)) => {
+							Self::transition(MigrationStage::FastUnstakeMigrationOngoing {
+								next_key: Some(next_key),
+							});
+						},
+						e => {
+							log::error!(target: LOG_TARGET, "Error while migrating fast unstake: {:?}", e);
```

Comment:
actually no need for error, since defensive is below. just would be nice to pass the `e` for defensive as well